scripts/new-upstream-snapshot

```diff
@@ -253,16 +257,22 @@ main() {
         command -v distro-info >/dev/null 2>&1 || fail "Install distro-info deb"
         # get ~20.10.1 for latest --stable release
         sru_ver_suff=$(distro-info --stable -r | awk '{printf "~%s.1", $1}')
+    elif [ "$first_devel_upload" = "true" ]; then
+        command -v distro-info >/dev/null 2>&1 || fail "Install distro-info deb"
+        # get ~20.10.1 for latest --stable release
+        sru_ver_suff=$(distro-info --devel -r | awk '{printf "~%s.1", $1}')
     else
         # if present, pull the '~16.04.x' off of the previous entry.
         sru_ver_suff=$(echo "$prev_pkg_ver" |
             sed 's,.*~\([0-9.]*\)[.][0-9]$,~\1.1,')
         [ "${sru_ver_suff}" = "${prev_pkg_ver}" ] && sru_ver_suff=""
     fi
+    echo "CHAD ${sru_ver_suff} ${first_devel_upload}"
```

Comment:
@blackboxsw did you mean to leave this line in?

Reply:
Whoopsie. cleared thanks.